src/script/descriptor.cpp

```diff
@@ -331,7 +323,8 @@ class ConstPubkeyProvider final : public PubkeyProvider
     }
     bool GetPrivKey(int pos, const SigningProvider& arg, CKey& key) const override
```

Comment:
Why does `GetPrivKey` of `ConstPubkeyProvider` not end up using the `pos` argument? Is it because it's supposed to be a "constant" provider?

Reply:
`pos` is really only used by `BIP32PubkeyProvider` to indicate child derivation index. Since no other `PubkeyProvider` can do derivation, it is ignored for all of them.